extract a little method name getMimeType

chengchaos opened the pull request:
when I used Apache Shiro and setting the "sessionManager" is "org.apache.shiro.web.session.mgt.DefaultWebSessionManager".
the request.getSession().getServletContext() method returned null.

chengchaos committed:
extract a little method name getMimeType

when I used Apache Shiro and setting the "sessionManager" is "org.apache.shiro.web.session.mgt.DefaultWebSessionManager".
the request.getSession().getServletContext() method returned null.

chengchaos committed:
Update WebjarsServlet.java

jamesward commented on src/main/java/org/webjars/servlet/WebjarsServlet.java:
```diff
@@ -29,7 +29,9 @@
-    private static final Logger logger = Logger.getLogger(WebjarsServlet.class.getName());
+	private static final long serialVersionUID = 1L;
+
+	private static final Logger logger = Logger.getLogger(WebjarsServlet.class.getName());
```
Nit picky but can you make sure you are using spaces, not tabs.

chengchaos replied:
I am sorry about use tabs, I am effort correct them.

chengchaos committed:
delete the getMimtType method

delete the getMimtType() method, because the Servlet has a getServletContext() method.

jamesward commented:
Thanks.  This is being released as `1.3` shortly.
